docs/core/tutorials/snippets/with-visual-studio/csharp/Program.cs

```diff
@@ -13,7 +13,7 @@ public static void Main(string[] args)
             Console.WriteLine("\nWhat is your name? ");
             var name = Console.ReadLine();
             var date = DateTime.Now;
-            Console.WriteLine($"\nHello, {name}, on {date:d} at {date:t}!");
+            Console.WriteLine($"{Environment.NewLine}Hello, {name}, on {date:d} at {date:t}!");
             Console.Write("\nPress any key to exit...");
```

Comment:
For consistency we should change the other two instances of \n to Environment.NewLine also.